src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -47,12 +49,20 @@
 public final class SetupUtils {
 
     private static final ScheduledExecutorService DEFAULT_SCHEDULED_EXECUTOR_SERVICE = ExecutorServiceHelpers.newScheduledThreadPool(3, "SetupUtils");
+    private static final String PRAVEGA_USERNAME = "admin";
```

Comment:
These file paths make me nervous because they presuppose the working directory of the test.   

Ideally we would not depend on the pravega submodule for the below files.  The best solution would be to place them into `src/test/resources` and extract them as temporary files.   

Could do something like:
```
import java.nio.file.Files;
public static Path copyResourceToFile(String resourceName) throws IOException {
    Path tempPath = Files.createTempFile("test-", ".tmp");
    tempPath.toFile().deleteOnExit();
    try (InputStream stream = ResourceManager.class.getResourceAsStream(jarFilePath)) {
        Files.copy(ResourceManager.class.getResourceAsStream(resourceName), tempPath, StandardCopyOption.REPLACE_EXISTING);
    }
    return tempPath;
}
```

Reply:
Done. Thanks for the tip! Somehow had trouble creating stream using ResourceManager. Used getClass().getClassLoader() instead.